src/popup/components/TransactionListItem.vue

```diff
@@ -154,19 +154,19 @@ export default defineComponent({
     });
 
     const fiatAmount = computed(() => {
-      const aeToken = tokens.value?.find((t) => t?.isAe);
-      if (!aeToken || isErrorTransaction.value || isDexPool.value) {
+      const protocolCoin = tokens.value?.find(({ assetType }) => assetType === ASSET_TYPES.coin);
```

Comment:
This check looks off for me. `tokens` should hold the list of tokens. Not coins. So the test `assetType === ASSET_TYPES.coin` for me is a workaround or broken logic.

Reply:
`tokens` should probably be renamed to `assets` since it seems that it was always holding all kinds of assets and not just tokens

Reply:
Good point: we need a task that would refactor the `useTransactionTokens` to `useTransactionAssets`. Can you create one?